### PR TITLE
Implement `Streamable` for `RxpReader`

### PR DESCRIPTION
**Pull request by gadomski**
As discussed in #2229.

Since there isn't CI support for this plugin, I'll note here that tests pass on my personal Ubuntu box.

### CODE REVIEW

File: plugins/rxp/io/RxpPointcloud.cpp [hunk 1]
```diff
@@ -79,39 +83,74 @@ RxpPointcloud::~RxpPointcloud()
 }
 
 
-point_count_t RxpPointcloud::read(PointViewPtr view, point_count_t count)
+bool RxpPointcloud::readOne(PointRef& point)
 {
-    point_count_t numRead = 0;
+    if (this->readSavedPoint(point)) {
```
Comment (abellgithub) [inline plugins/rxp/io/RxpPointcloud.cpp]: Would think `this` would be superfluous in this context.

File: plugins/rxp/io/RxpPointcloud.cpp [hunk 2]
```diff
@@ -52,24 +52,28 @@ Dimension::Id getTimeDimensionId(bool syncToPps)
...
-    , m_view(new PointView(table))
-    , m_idx(0)
     , m_syncToPps(syncToPps)
-    , m_minimal(minimal)
     , m_reflectanceAsIntensity(reflectanceAsIntensity)
     , m_minReflectance(minReflectance)
     , m_maxReflectance(maxReflectance)
     , m_rc(scanlib::basic_rconnection::create(uri))
     , m_dec(m_rc)
+    , m_points()
```
Comment (abellgithub) [inline plugins/rxp/io/RxpPointcloud.cpp]: Unnecessary.

File: plugins/rxp/io/RxpPointcloud.cpp (hunk 1 above)
Comment (gadomski) [reply plugins/rxp/io/RxpPointcloud.cpp]: It's not technically necessary, but I like to use `this` to disambiguate the call from a free function call. Happy to remove if you'd like -- if so, maybe worth putting into the style guide as a recommended practice ("don't use `this` for method calls")?

File: plugins/rxp/io/RxpPointcloud.cpp (hunk 2 above)
Comment (gadomski) [reply plugins/rxp/io/RxpPointcloud.cpp]: Habit carry-over from "you must initialize everything" languages. Same as above comment, happy to remove if you'd like, but recommend putting something in the style guide about it to help prevent future churn?

File: plugins/rxp/io/RxpPointcloud.cpp (hunk 1 above)
Comment (abellgithub) [reply plugins/rxp/io/RxpPointcloud.cpp]: The style guide:

"Make your code look like existing code."

Of course, people are free to interpret this as they wish.

Comment (gadomski) [reply plugins/rxp/io/RxpPointcloud.cpp]: Code updated via force-push per both comments.

**Pull request merged**